INSTALL.md

```diff
@@ -77,3 +77,16 @@ Please see [INSTALL.make.md](dev/doc/INSTALL.make.md) for build and
 installation instructions using `make`. If you wish to experiment with
 the Dune-based system see the [dune guide for
 developers](dev/doc/build-system.dune.md).
+
+Notes on build-time dependencies
```

Comment:
Not sure "build-time dependencies" is the right wording here; already above we discuss about what we need to build, and just below we talk about run-time dependencies.

I suggest:
- we move the native dependencies to its own section, "Dependencies of native compilation"
- we remove the second bullet, and make it into a section "OCaml toolchain advisory" , also I would just start with "Both Coq plugins and vo files require at loading time the same exact toolchain setup that was used to compile them, etc..."

Other than that, I'd like to merge this before the branch date.